src/pathfinding_service/service.py

```diff
@@ -253,3 +275,13 @@ def create_token_network_for_address(
         )
         token_network_listener.start()
         self.token_network_listeners.append(token_network_listener)
+
+    def handle_message(self, message: SignedMessage):
+        if isinstance(message, UpdatePFS):
+            self.on_pfs_update(message, UpdatePFS)
+            # TODO validate message
+        else:
+            log.info('Ignoring unknown message type')
+
+    def on_pfs_update(self, message, UpdatePFS):
```

Comment:
this should be `message: UpdatePFS`, right?

Reply:
yes, changed it